src/js/player.js

```diff
@@ -1927,6 +1934,11 @@ class Player extends Component {
             this.play();
           }
 
+          this.trigger('sourcechange', {
```

Comment:
should this be wrapped in a setTimeout like mentioned in https://github.com/videojs/video.js/issues/2382? The source loading algorithm is asynchronous, so, if someone listens to `sourcechange` and calls `player.currentSrc()` they may not get the latest value.